### PR TITLE
MOTECH-772 documenting the OpenMRS module

### PR DESCRIPTION
**Pull request by craigappl**

### CODE REVIEW

File: docs/source/openmrs.rst [hunk 1]
```diff
@@ -4,6 +4,133 @@
...
+
+- Create Encounter
+    This task action allows you to create patient encounters in OpenMRS. The task uses the POST command to the OpenMRS Rest Web Services module. Below is a description of each field in the action.
+
+    +-------------------------+------------------------------------------------------------+
+    |Field                    |Description                                                 |
+    +=========================+============================================================+
+    |Encounter Date           |The date and time the encounter was created.                |
+    +-------------------------+------------------------------------------------------------+
+    |Encounter Type           |The UUID of the encounter type                              |
```
Comment (kmadej) [inline docs/source/openmrs.rst]: Now in our OpenMRS module the encounter type field in create encounter action is a string representing name of encounter type

**Comment by craigappl** [comment]
@kmadej  I updated the doc and just pushed the changes.

File: docs/source/openmrs.rst [hunk 2]
```diff
@@ -4,6 +4,133 @@
...
+
+- Create Encounter
+    This task action allows you to create patient encounters in OpenMRS. The task uses the POST command to the OpenMRS Rest Web Services module. Below is a description of each field in the action.
+
+    +-------------------------+------------------------------------------------------------+
+    |Field                    |Description                                                 |
+    +=========================+============================================================+
+    |Encounter Date           |The date and time the encounter was created.                |
+    +-------------------------+------------------------------------------------------------+
+    |Encounter Type           |The name of the OpenMRS encounter type.                      |
```
Comment (sebbrudzinski) [inline docs/source/openmrs.rst]: The table is broken

**Comment by craigappl** [comment]
Removed a space. Sorry I didn't catch that the first time.

**Pull request merged**